tests/unit/memoizers/presentation-memoizer-test.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Unit\Memoizers;
+
+use Mockery;
+use Yoast\WP\SEO\Context\Presentation;
+use Yoast\WP\SEO\Presentations\Indexable_Presentation;
+use Yoast\WP\SEO\Tests\Unit\Doubles\Context\Meta_Tags_Context_Mock;
+use Yoast\WP\SEO\Tests\Unit\Doubles\Context\Presentation_Mock;
+use Yoast\WP\SEO\Tests\Unit\Doubles\Memoizers\Presentation_Memoizer_Double;
+use Yoast\WP\SEO\Tests\Unit\Doubles\Models\Indexable_Mock;
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+use YoastSEO_Vendor\Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class Presentation_Memoizer_Test.
+ *
+ * @coversDefaultClass \Yoast\WP\SEO\Memoizers\Presentation_Memoizer
+ *
+ * @group memoizers
+ */
+class Presentation_Memoizer_Test extends TestCase {
+
+	/**
+	 * The service container.
+	 *
+	 * @var ContainerInterface
+	 */
+	protected $container;
```

Comment:
I wonder why the Presentation_Memoizer needs the DI container?

Reply:
The Presentation_Memoizer has the container in its constructor. That's why I also needed it in the test. Maybe the Presentation_Memoizer itself can be set up differently, but that's outside the scope of this PR / my expertise :)